wagtail/admin/ui/side_panels.py

```diff
@@ -55,13 +56,17 @@ def __init__(
         self.locking_enabled = isinstance(self.object, LockableMixin)
 
     def get_status_templates(self, context):
-        templates = ["wagtailadmin/shared/side_panels/includes/status/workflow.html"]
+        includes_dir = "wagtailadmin/shared/side_panels/includes/status/"
```

Comment:
I think we should stick to writing out template paths in full every time unless there's a very good reason not to - it's common to grep the codebase for a template path to find out where it's being used.

Reply:
Ah yep, that makes sense. Thanks! Updated.